changeLog.md

```diff
@@ -0,0 +1,2 @@
+# v 0.1.1
```

Comment:
What are your thoughts about using the calVer format?
https://wiki.freecad.org/Package_Metadata#%3Cversion%3E

Reply:
Let's use the calVer format with a sequence number.  That way the versions can be automatically calculated.  If we use major.minor or some other variant, then a human has to decide when a version is major.
So addon versions will look like this 2023.07.06.01  (YYYY.MM.DD.SEQ)

Reply:
Sounds good to me.